.travis.yml

```diff
@@ -0,0 +1,173 @@
+# Che-Theia workflow
+
+dist: focal
+os: linux
+cache: false
+git:
+  depth: false 
+
+language: node_js
+node_js: "12"
+
+install:
+  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+  - docker login quay.io -u="$QUAY_USERNAME " -p="$QUAY_PASSWORD"
+
+env: 
+  global:
+  - TAG=next
```

Comment:
I think the tag would conflict there with current one as we want to experiment for a while how it goes ?

Reply:
is that why we have the TRAVIS_TAG to append on for now, and later set to "" ?

Reply:
> I think the tag would conflict there with current one as we want to experiment for a while how it goes ?

TAG won't conflict with the current ` next ` tag as while publishing the image we have suffixed the next tag with TRAVIS_TAG (i.e. `next-travis`). 

> is that why we have the TRAVIS_TAG to append on for now, and later set to "" ?

Yes. TRAVIS_TAG is introduced as Travis is experimental. Later this flag can be removed completely from all files.

Reply:
I still don't get why we do need an addition TRAVIS_TAG variable
if TAG is set to `next-travis` then image name will be suffixed by this tag so it should be enough to change the suffix